nextcord/message.py

```diff
@@ -1231,6 +1235,10 @@ async def edit(
         attachments: List[:class:`Attachment`]
             A list of attachments to keep in the message. If ``[]`` is passed
             then all attachments are removed.
```

Comment:
In v10, even if `[]` is not passed, all attachments will be removed.

```suggestion
            A list of attachments to keep in the message.
```